File: Day10/Solution.cpp
int main(){
    int n;
    cin >> n;
    int max = 0;
    int count = 0;

    while(n>0)
    {
        if(n%2==1)
            count++;
        else
            count = 0;
        if(max<count)
            max = count;
        n=n/2;
    }
    cout<<max;
    return 0;
}

File: Day10/Solution.go
package main
import "fmt"

var n int
var max int = 0
var count int = 0

func main(){
  fmt.Scan(&n)
  
  for n>0{
      if(n%2 == 1){
          count++
      } else{
          count = 0
      }
      
      if(max<count){
          max = count
      }  
      n/=2
  }
  
  fmt.Print(max)
}
